client/reader/full-post/controller.js

```diff
@@ -69,7 +69,6 @@ export function blogPost( context ) {
 				blogId: blogId,
 				postId: postId,
 				context: context,
-				setPageTitle: setPageTitle,
```

Comment:
There are two other controller functions which render components passing `setPageTitle` that I suspect need to be removed:

- `blogPostNew`
- `feedPostNew`